src/qililab/waveforms/iq_pair.py

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""IQPair dataclass."""
 from dataclasses import dataclass
 
 from qililab.waveforms.waveform import Waveform
 
 
 @dataclass
 class IQPair:  # pylint: disable=missing-class-docstring
+    """IQPair dataclass, containing the real (I) and imaginary (Q) parts of a signal."""
```

Comment:
I am not really confortable with the 'imaginary' and 'real' part descriptions for I and Q after reading about it, I would rather stick to the 'in-phase' and 'quadrature' descriptions, respectively. What do you think?

Reply:
Sure yes! It's not really my domain, so I'll go with what you think its best!

Reply:
Changed!